cppcache/src/TcpConn.cpp

```diff
@@ -113,16 +113,11 @@ TcpConn::TcpConn(const std::string ipaddr,
 TcpConn::TcpConn(const std::string host, uint16_t port,
                  std::chrono::microseconds timeout, int32_t maxBuffSizePool)
     : socket_{io_context_} {
-  auto beforeResolvePoint = std::chrono::system_clock::now();
-  auto results = resolve(host, port, timeout);
-  auto elapsedTime = std::chrono::duration<double, std::micro>(
-      std::chrono::system_clock::now() - beforeResolvePoint);
+  auto results = resolve(host, port);
```

Comment:
I believe it is in intentional that the resolve of the host is also part of the connection timeout. Now it looks like resolve could take an indeterminate amount of time followed by only connect limited by timeout.

Reply:
Ok, looking at the original organization of the code using ACE the resolve step was not included in the timeout. The resolve happened on the construction of the `TcpConn` when it initialized the `ACE_INET_Addr` member. The timeout was only used for the connect phase. This change seems consistent with that change.